public/redux/reducers/elasticsearch.ts

```diff
@@ -204,6 +276,24 @@ export const searchES = (requestData: any): APIAction => ({
     client.post(`..${AD_NODE_API._SEARCH}`, requestData),
 });
 
+export const createIndex = (indexConfig: any): APIAction => ({
+  type: CREATE_INDEX,
+  request: (client: IHttpService) =>
+    client.put(`..${AD_NODE_API.CREATE_INDEX}`, { indexConfig }),
+});
+
+export const bulk = (body: any): APIAction => ({
+  type: BULK,
+  request: (client: IHttpService) =>
+    client.post(`..${AD_NODE_API.BULK}`, { body }),
+});
+
+export const deleteIndex = (index: string): APIAction => ({
```

Comment:
So currently don't support this in the UI actually. Had originally added this to support deletion, but for the current implementation we decided not to do add this on the UI. Main reason being it could get really complex and confusing for a user to know what would be deleted, and could cause errors if a user changes the name of the existing sample detector, or creates other detectors on top of the sample index, etc. Detector deletion can always be done via detector list or detector details pages, and index deletion via console.

Reply:
Note that this will probably be a future improvement.